python/samples/kernel-syntax-examples/setup_logging.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft. All rights reserved.
+import logging
+
+import semantic_kernel as sk
+from semantic_kernel.connectors.ai.open_ai import OpenAIChatCompletion
+
+
+def setup_logging():
```

Comment:
I would like to see this function, without the last line in a utils.log.py file, and this sample loading and calling it, then setting the default loglevel in code. The format we can be opiniated about while the loglevel is common enough that most python devs know how to set it or use regular structures for dealing with it.

Reply:
Good call, fixed